.env.example

```diff
@@ -0,0 +1,10 @@
+MONGO_URI=mongodb+srv://LVien:<password>@practicum.twrhx3o.mongodb.net/databasename?retryWrites=true&w=majority
```

Comment:
LVien is the username here right? Can you write it similar to how you did for password (<username>). Similarly I would recommend hiding practicum.twrhx3o.mongodb.net as well.

Reply:
Noted. I fixed the code as below : 
`MONGO_URI=mongodb+srv://<username>:<password>@xxxxxx.xxxxx.mongodb.net/database name?retryWrites=true&w=majority
("url from mongodb connection")`